Yank/yank.py

```diff
@@ -316,12 +316,38 @@ def _create_phase(self, phase, reference_system, positions, atom_indices, thermo
         # Make a deep copy of the reference system so we don't accidentally modify it.
         reference_system = copy.deepcopy(reference_system)
 
+        # Store a serialized copy of the reference system.
+        from simtk.openmm import XmlSerializer
+        metadata['reference_sysem'] = XmlSerializer.serialize(reference_system)
```

Comment:
The fully-interacting system is the one we want to compute the reference free energy to, but it may have an enlarged cutoff, or might be perturbed in some other way. We probably also want the original `System` object (the unmodified reference system) in the NetCDF file as well.